### PR TITLE
Update plugins-known-issues.md

### PR DESCRIPTION
**Pull request by ravenloveriel**
I’m seeing that this part of the wordfence issue of our doc might be misleading when the customers are creating their symlinks. Since the symlink commands in the doc are assuming that our customers are in their /code directory and not in /code/wp-content directory.

In step 3, the code snippet for that is correct. However, stating /wp-content is a bit misleading.

Closes #

## Summary

**[Doc Page Title](https://docs.pantheon.io/doc-title)** - <Enter a one sentence summation of the pertinent changes (including not-yet-completed work) provided by this PR.>

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by whitneymeredith** [comment]
[Preview](https://pr-8550-documentation.appa.pantheon.site/plugins-known-issues#wordfence)

**Comment by whitneymeredith** [comment]
@scottbuscemi Can you take a look at these changes when you get a chance?

**Comment by scottbuscemi** [comment]
@whitneymeredith Hi! I'll share this in the Slack to have some CMS Eco folks check this out.

**Review by namespacebrian — APPROVED** [review]

File: source/content/plugins-known-issues.md [hunk 1]
```diff
@@ -1043,6 +1049,10 @@ export ENV=dev
   ln -s ../../files/private/wflogs ./wp-content/wflogs
   ln -s ../files/private/wordfence-waf.php ./wordfence-waf.php
   ln -s ../files/private/.user.ini ./.user.ini
+
+  ![replicate-1](https://github.com/pantheon-systems/documentation/assets/103234849/5b7e8d70-e081-49c8-a71d-09a9f8d66109)
+  ![replicate-2](https://github.com/pantheon-systems/documentation/assets/103234849/72b71341-ed1b-4692-b501-4c4eace7ee41)
+  ![wordfence](https://github.com/pantheon-systems/documentation/assets/103234849/d1ae1e49-025a-4ef6-a8cd-8dbb5fdb0745)
```
Comment (namespacebrian) [inline source/content/plugins-known-issues.md:1055]: These 3 images are showing as part of the code block as if they were commands for the customer to type.. I'm pretty sure this is not intended..

image

**Review by namespacebrian — DISMISSED** [review]

File: source/content/plugins-known-issues.md (hunk 1 above)
Comment (CdrMarks) [reply source/content/plugins-known-issues.md]: @whitneymeredith despite your [commit](https://github.com/pantheon-systems/documentation/pull/8550/commits/65295f1603bd29a79bb989b45ca099bb4a38fea9) and the build that I see in the main conversation, I'm still seeing what Brian reported on the preview site.

Comment (whitneymeredith) [reply source/content/plugins-known-issues.md]: @CdrMarks the code for the images doesn't show on the preview link or on my local. can you tell me where you're seeing them?

**Review by rachelwhitton — APPROVED** [review]

**Pull request merged**